user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -772,51 +781,183 @@ private void onExperimentalGroupEditConfirmed(
+          log.debug("Updated experimental groups for project" + projectId);
+        });
+  }
+
+  private void displayFailedExperimentalGroupUpdate() {
+    getUI().ifPresent(ui -> ui.access(() -> {
+      messageSourceNotificationFactory.toast("experimental.groups.updated.failed", new Object[]{},
+          getLocale()).open();
+    }));
+  }
+
+  private void displaySuccessfulExperimentalGroupUpdate() {
+    getUI().ifPresent(ui -> ui.access(() -> {
+      messageSourceNotificationFactory.toast("experimental.groups.updated.success", new Object[]{},
+          getLocale()).open();
+    }));
+  }
+
+  private AsyncProjectService.VariableLevel toApi(VariableLevel level) {
+    return new AsyncProjectService.VariableLevel(level.variableName().value(),
+        level.experimentalValue().value(), level.experimentalValue().unit().orElse(null));
+  }
+
+  private AsyncProjectService.ExperimentalGroup toApi(ExperimentalGroupContent experimentalGroup) {
+    return new AsyncProjectService.ExperimentalGroup(experimentalGroup.id(),
+        experimentalGroup.groupNumber(),
+        experimentalGroup.name(), experimentalGroup.size(),
+        experimentalGroup.variableLevels().stream().map(this::toApi).collect(Collectors.toSet()));
+  }
+
   private void addExperimentalGroups(
       Collection<ExperimentalGroupContent> experimentalGroupContents) {
-    List<ExperimentalGroupDTO> experimentalGroupDTOS = experimentalGroupContents.stream()
-        .map(this::toExperimentalGroupDTO).toList();
+    if (experimentalGroupContents.isEmpty()) {
+      return;
+    }
+    var experimentalGroups = experimentalGroupContents.stream().map(this::toApi).toList();
     ExperimentId experimentId = context.experimentId().orElseThrow();
-    experimentInformationService.updateExperimentalGroupsOfExperiment(
-        context.projectId().orElseThrow().value(),
-        experimentId, experimentalGroupDTOS);
+    var projectId = context.projectId().orElseThrow();
+
+    var serviceCalls = new ArrayList<Mono<ExperimentalGroupCreationResponse>>();
+
+    experimentalGroups.forEach(experimentalGroup -> {
+      serviceCalls.add(
+          asyncProjectService.create(new ExperimentalGroupCreationRequest(projectId.value(),
+              experimentId.value(), experimentalGroup)));
+    });
+
+    Mono.when(serviceCalls).doOnSuccess(s -> {
+          displaySuccessfulExperimentalGroupCreation();
+          reloadExperimentalGroups();
+          showSampleRegistrationPossibleNotification();
+        }).doOnError(e -> {
+          log.error("Error while creating experimental group", e);
+          displayFailedExperimentalGroupCreation();
+        })
+        .subscribe(it -> {
+          log.debug("Added experimental groups for project" + projectId);
+        });
+  }
+
+  private void displayFailedExperimentalGroupCreation() {
+    getUI().ifPresent(ui -> ui.access(() -> {
+      messageSourceNotificationFactory.toast("experimental.groups.created.failed", new Object[]{},
```

Comment:
NIT: You can use MessageSourceNotificationFactory.EMPTY_PARAMETERS for empty parameters if you want.
```suggestion
      messageSourceNotificationFactory.toast("experimental.groups.created.failed", EMPTY_PARAMETERS,
```